linode/manager
# refactor: [M3-6903] - Replace Select with Autocomplete in: volumes

## Description 📝
This PR eliminates redundant logic and reuses ConfigSelect in AttachVolumeDrawer. 
Replaced `Select` with Autocomplete in `ConfigSelect` component.
Reused `ConfigSelect` instead of Select in `AttachVolumeDrawer` component. 

## Target release date 🗓️
6/10



## How to test 🧪

### Verification steps
(How to verify changes)
- Navigate to http://localhost:3000/volumes and click on attach menu option.
- Verify there is no regression in volume attach / detaching behaviors. 
- Verify there is no regression in Volume create flow.
- Navigate to http://localhost:3000/linodes/< Linode_id>/storage - Verify there is no regression in  creating / attaching / detaching volumes. 

## As an Author I have considered 🤔

*Check all that apply*
